rust/private/test_runner_template.rs

```diff
@@ -0,0 +1,50 @@
+use std::vec::Vec;
+use std::collections::BTreeMap;
+use std::process::Command;
+
+#[cfg(target_family = "unix")]
+use std::os::unix::process::CommandExt;
+
+/// This is a templated function for defining a map of environment
+/// variables. The comment in this function is replaced by the
+/// definition of this map.
+fn environ() -> BTreeMap<&'static str, &'static str> {
+// {environ}
+}
+
+/// Parse the command line arguments but skip the first element
+fn args() -> Vec<String> {
+    std::env::args().skip(1).collect()
+}
+
+/// Simply replace the current process with our test
+#[cfg(target_family = "unix")]
+fn exec(environ: BTreeMap<&'static str, &'static str>) {
+    Command::new("{executable}")
+    .envs(environ.iter())
+    .args(args())
+    .exec();
+}
+
+/// On windows, there is no way to replace the current process
+/// so instead we allow the command to run in a subprocess.
+#[cfg(target_family = "windows")]
+fn exec(environ: BTreeMap<&'static str, &'static str>) {
+    let output = Command::new("{executable}")
```

Comment:
```suggestion
    let output = Command::new(r#####\"{executable}\"#####)
```

Reply:
I figured this and the environment variable would be safe but I'm happy to be consistent 😄